README.md

```diff
@@ -7,41 +7,43 @@ The data lineages and data quality are tracked automatically between HTTP endpoi
 By default, the `Jaeger backend tracer + Zipkin reporter` is used to keep the spans published to a observability platform, this tracer is combined with the Kensu tracer to collect what is important for Kensu to build lineage and stats.
 
 
-Build and run
-------
-
-Starting Jax-Rs server for sample app (incl Wildflow & MySQL in docker):
-```bash
-( cd ./docker-dir && run-docker.sh )
-./build-dependencies.sh
-./build-and-run-app.sh
-```
-
+One-click run all in Docker
+-----
 
-Some endpoints working out of the box (enough to have just MySQL):
+First run `./one-click-start-all-prebuilt.sh` (to start this app with MySQL & Zipkin inside a docker), and, after waiting a few minutes for it to start up, visit some of these endpoints:
 
 - [http://127.0.0.1:8080/rest/v1/product-line/Motorcycles](http://127.0.0.1:8080/rest/v1/product-line/Motorcycles)
 - [http://127.0.0.1:8080/rest/v1/product-line/Ships](http://127.0.0.1:8080/rest/v1/product-line/Ships)
 - [http://127.0.0.1:8080/rest/v1/order-details/product-line/Ships?maxResults=17](http://127.0.0.1:8080/rest/v1/order-details/product-line/Ships?maxResults=17)
 
-Check on Kensu UI: data catalog, lineages, and projects.
+This will trigger some metadata on data movement and data quality getting collected and sent to Zipkin and/or Kensu servers.
 
+### Check results: Zipkin & Kensu app
 
-Debugging
------
+If you have access to Kensu App, on its UI you may first check - data catalog, lineages, and projects.
```

Comment:
do we want to at least include a screenshot of Kensu UI? pros/cons? @michele-pinto-kensu